pset-iterators.js

```diff
@@ -137,7 +165,13 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 5:')
 
 // Add your code below this line
+const livingPresidents = presidents.filter((president) => {
+  let alive = president.death_year === null
+      return alive
+  })
 
+
+  console.log(livingPresidents)
```

Comment:
@jct117 Good job on this solution using the filter method to look up the current living presidents.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L148)